README.md

```diff
@@ -14,6 +14,12 @@ Commerce. The Department of Commerce seal and logo, or the seal and logo of
 a DOC bureau, shall not be used in any manner to imply endorsement of any
 commercial product or activity by DOC or the United States Government.
 ```
+```
+This project contains derivative works from the Geophysical Fluid Dynamics
+Laboratory GitHub projects for the FV3 dynamical core.  As such, licensing
```

Comment:
Recommend you make "Geophysical Fluid Dynamics Laboratory" and "FV3 dynamical core" hyperlinks to http://www.gfdl.noaa.gov/ and https://www.github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere, respectively. That will allow others to quickly navigate to the origins of this work.

Reply:
Lynd, I'm not sure if you know or not, but in MD you include the text you'd like to link within `[` & `]`, following the [ ],   you insert the hyperlink into `(` `)` immediately afterwards. This makes it look very clean. See blow for an example: 
[Geophysical Fluid Dynamics Laboratory](http://www.gfdl.noaa.gov/)
[FV3 dynamical core](https://www.github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere)